src/components/form/examples/custom-error-message-form.tsx

```diff
@@ -0,0 +1,67 @@
+import { ValidationStatus, FormError } from '@limetech/lime-elements';
```

Comment:
Small feedback on the commit messages:

### First commit:

> feat: add transformErrors prop

- add a "scope" when applicable: `feat(form): …`
This can sometimes be a little arbitrary, but here in Lime Elements it's almost always super easy, because we use the component's name, minus the `limel-` prefix. In other repos it's often a bit more complicated, since we tend do have deeper structures, and you might want to keep the "scope" pretty short.
- write the commit message subject so it makes sense for someone reading the changelog
The first line of the commit message (`type(scope): subject`) is used to create the changelog entry, so use a scope and a subject that makes sense in that context. Try to summarise what has been done from a user's perspective when possible. The first line can be up to 100 characters, including the type and scope, so if it fits you can also include something about what problem it fixes. In this case, you already have a solid summary in the commit message body, although I'd actually skip the "provided by JSON schema" part: `feat(form): make it possible to customize the default error messages` 👌
- include `fix: #123` at the bottom of the commit message (if the commit fixes an issue)
That way, the changelog item for this commit will include a link directly to the issue it fixes, like for the second item here: <img width="722" alt="image" src="https://user-images.githubusercontent.com/154725/144404904-5e72a53c-c510-4547-9bbd-4ef622c00f49.png">

---

### The last three commits

The last three commits should just be fixup commits, since they fix things in a commit that hasn't been merged to `next. Fixup commits indicate that they should be squashed into an existing commit. We also have a PR check that fails if there are any fixup commits. That lets us make any changes based on review feedback with fixup commits, that remain un-squashed while the review is ingoing (so we can review only the changes each time something new is pushed), while the PR check prevents us from accidentally merging the PR before squashing the fixups. [Here's a good post](https://thoughtbot.com/blog/autosquashing-git-commits) explaining fixup commits better than I can.

---

I can help you out with editing the existing commits, if you want me to 👍

Reply:
Thank you for the feedback @adrianschmidt, it all makes sense! I will review the guide you linked and conform to these rules in future PRs!

Reply:
Yes, some help with editing the existing commits would be great!

Reply:
Fixed! Since the commit history was the last thing to fix before I'd approve and merge this, I realised it was easiest to just squash the commits while I was at it.

(Just to clarify: we don't just blindly squash each PR to _one_ commit. We try to make [atomic commits](https://github.com/Lundalogik/lime-elements/blob/next/CONTRIBUTING.md#atomic-commits), and consider tests, docs, and examples for a new feature as the same "atom" as the feature itself. So in this case, a single commit 🙂)